microarray/src/org/labkey/microarray/MicroarrayFolderType.java

```diff
@@ -35,13 +35,10 @@ public MicroarrayFolderType(MicroarrayModule module)
         super("Microarray",

```

Comment:
I don't think this folder type is useful anymore. I suggest just deleting it. The assay folder type provides almost the same setup.

Reply:
Deleted!